README.md

```diff
@@ -35,7 +35,9 @@ See the [Autotune Architecture](/design/README.md) for more details on the archi
 
 ## Contributing
 
-We welcome your contributions! See [CONTRIBUTING.md](/CONTRIBUTING.md) for more details.
+We welcome your contributions! See [CONTRIBUTING.md](/CONTRIBUTING.md) for more details. 
+
+Join the [Kruize Slack](http://kruizeworkspace.slack.com/) to connect with the community, ask questions, and collaborate!
```

Comment:
@bhanvimenghani can you please add the actual link to sends a request to be added (https://join.slack.com/t/kruizeworkspace/shared_invite/zt-nmxjz4n6-xX1FiCUljciMzltcItDI~w)

Reply:
Also I think it would be better to add a QR code, than just the link

Reply:
This link does not expire (You can select that as an option when creating it, which is what I did)

Reply:
added the QR code